site/content/docs/cli/using-environment-variables.md

```diff
@@ -0,0 +1,93 @@
+
+Use local environment variables to inject or replace values in your `Check`, `AlertChannel` or other constructs at 
+**build time** when the CLI compiles your constructs for testing and deploying.
+
+Here is an example of setting up an `SmsAlertChannel` where we pass in the actual phone number from a local environment
+variable. Note the exclamation mark `!` at the end. This is to tell the Typescript compiler the value will be set.
+Alternatively you can use a string template.
+
+```ts
+import { SmsAlertChannel } from '@checkly/cli/constructs'
+
+export const smsChannel = new SmsAlertChannel('sms-channel-1', {
+  phoneNumber: process.env.PHONE_NUMBER!
+  // phoneNumber: `${process.env.PHONE_NUMBER}` this also works
+})
+```
+Now, when you execute `test` or `deploy` the variable needs to part of the parent process `env` object. This is typically
+done by just prepending the variable to the command.
+
+```
+PHONE_NUMBER=0987654321 npx checkly deploy
+```
+
+The above command will replace `process.env.PHONE_NUMBER` with the actual number and persist that as an SMS alert channel
+in your Checkly account.
+
+## Remote Environment Variables
+
+Checkly also stores environment variables in your Checkly account. These can exist at the 
+[Account, Group or Check level](/docs/browser-checks/variables/). Here is what you need to remember:
+
+1. Use remote environment variables to dynamically inject or replace values during runtime of a check.
+2. Remote variables can be set and overridden when invoking the `test` command.
+
+You will typically use remote environment variables inside the code dependencies you write, e.g. Playwright tests, setup
+ and teardown scripts. The point is that the `process.env.SOME_VARIABLE` stays in your code and are only interpreted when
+a check executes on the Checkly cloud.
+
+Here is an example of a Playwright script using an `ENVIRONMENT_URL` variable to define the page the visit. We also added
+a fallback value in case that variable is not defined for some reason.
+
+```ts
+import { test } from '@playwright/test'
+
+test('Check Home Page', async ({ page }) => {
+  const response = await page.goto(process.env.ENVIRONMENT_URL! || 'https://wwww.checklyhq.com')
+  await page.screenshot({ path: 'home.jpg' })
+})
+```
+
+You can now test this check and temporarily set the environment variable as follows. 
+
+```
+npx checkly test -e ENVIRONMENT_URL="https://staging.checklyhq.com"
+```
+
+- Notice that we pass in the variable using the `-e` flag. This means it will be passed to the cloud environment and made 
+available during runtime.
+- After deploying this check, either the `ENVIRONMENT_URL` is set at the Account, Group or Check level or in the example's
+case the fallback URL is used.
+- Prepending the variable like `ENVIRONMENT_URL="https://staging.checklyhq.com" npx checkly test` has no effect as local
```

Comment:
Maybe we can rephrase one this or the sentence above for clarity.

> The above command will replace `process.env.PHONE_NUMBER` with the actual number and persist that as an SMS alert channel in your Checkly account.

Reply:
I did some small rewording on this.